Firestore/core/src/nanopb/message.h

```diff
@@ -188,8 +188,8 @@ class Message {
 };
 
 /**
- * A wrapper of const Message objects that facilitates shared ownership of
- * immutable Protobuf data.
+ * A wrapper of Message objects that facilitates shared ownership of Protobuf
```

Comment:
It's a little unfortunate that this class is no longer immutable. How about adding a TODO to create a `template <typename const T>` specialization?

Reply:
Done